tailwind.config.js

```diff
@@ -57,9 +57,10 @@ module.exports = {
       },
       screens: {
         xxs: '17.6rem',
-        xs: '22.5rem',
+        xs: '22.6rem',
```

Comment:
Why do we need this change? Won't it break smt in the other place?

Reply:
We need it to optimize apply styles on 360px as well (not just 359px)
![image](https://user-images.githubusercontent.com/49658988/188866692-52c764af-ee1b-48c8-a0b5-b9349f063722.png)

Reply:
I double-checked, all pages work fine both on <360px and >360px